package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Provides an API layer over Instagram's public API.",
   "main": "app.js",
   "scripts": {
-    "start": "NODE_ENV=prod node app.js",
+    "start": "set NODE_ENV=prod && node app.js",
```

Comment:
this will break on linux and heroku. We can use something like cross-env but since this does app does not depend on any external node modules, I'll like to keep it that way.

Any other suggestions?